playground/fs-serve/__tests__/fs-serve.spec.ts

```diff
@@ -43,14 +43,12 @@ describe.runIf(isServe)('main', () => {
   })
 
   test('unsafe fetch with special characters (#8498)', async () => {
-    expect(await page.textContent('.unsafe-fetch-8498')).toMatch(
-      '403 Restricted',
-    )
-    expect(await page.textContent('.unsafe-fetch-8498-status')).toBe('403')
+    expect(await page.textContent('.unsafe-fetch-8498')).toBe('')
+    expect(await page.textContent('.unsafe-fetch-8498-status')).toBe('404')
   })
 
   test('unsafe fetch with special characters 2 (#8498)', async () => {
-    expect(await page.textContent('.unsafe-fetch-8498-2')).toMatch('')
+    expect(await page.textContent('.unsafe-fetch-8498-2')).toBe('')
```

Comment:
IIUC This line wasn't testing anything. 😅 `.toMatch('')` returns always true.
https://stackblitz.com/edit/vitest-dev-vitest-byvrxk?file=test%2Fbasic.test.ts